docs/core/releases-and-support.md

```diff
@@ -1,109 +1,73 @@
+New major .NET versions are released annually in November, at [.NET Conf](https://www.dotnetconf.net/).
 
-### Runtime roll-forward and compatibility
-
-Major and minor updates install side by side with previous versions. An application built to target a specific *major.minor* version continues to use that targeted runtime even if a newer version is installed. The app doesn't automatically roll forward to use a newer *major.minor* version of the runtime unless you opt in for this behavior. An application that was built to target .NET Core 3.0 doesn't automatically start running on .NET Core 3.1. We recommend rebuilding the app and testing against a newer major or minor runtime version before deploying to production. For more information, see [Framework-dependent apps roll forward](versions/selection.md#framework-dependent-apps-roll-forward) and [Self-contained deployment runtime roll forward](deploying/runtime-patch-selection.md).
-
-Servicing updates are treated differently from major and minor releases. An application built to target .NET 5.0.0 runs on the 5.0.0 runtime by default. It automatically rolls forward to use a newer 5.0.1 runtime when that servicing update is installed. This behavior is the default because we want security fixes to be used as soon as they're installed without any other action needed. You can opt out from this default roll forward behavior.
-
-## .NET version lifecycles
-
-.NET Core, .NET 5, and later versions adopt the [modern lifecycle](/lifecycle/policies/modern) rather than the [fixed lifecycle](/lifecycle/policies/fixed) that has been used for .NET Framework releases. Products with fixed lifecycles provide a long fixed period of support, for example, 5 years of mainstream support and another 5 years of extended support. Mainstream support includes security and non-security fixes, while extended support provides security fixes only. Products that adopt a modern lifecycle have a more service-like support model, with shorter support periods and more frequent releases.
-
-### Release tracks
-
-There are two support tracks for releases:
+Patch updates are released monthly on the second Tuesday of each month, also known as Patch Tuesday.
 
-* *Current* releases
+Minor versions of the .NET SDK are released approximately quarterly. These are known as [feature bands](#feature-bands-sdk-only).
 
-  These versions are supported until six months after the next major or minor release ships. Previously (.NET Core 3.0 and earlier), these releases were supported for only three months after the next major or minor release shipped.
-
-  Example:
+## Release types
 
-  * .NET Core 3.0 shipped in September  2019 and was followed by .NET Core 3.1 in December 2019.
-  * .NET Core 3.0 support ended in March 2020, 3 months after 3.1 shipped.
+Each .NET release is defined as either  **Short Term Support (STS)** or **Long Term Support (LTS)**, at the beginning of the release.
 
-* *Long Term Support* (LTS) releases
+The release types:
```

Comment:
I think in here we shouldn't introduce the release type names ahead of their definition. Something like this would look better:
![image](https://user-images.githubusercontent.com/12971179/182269376-28ac616a-017e-4f00-9e59-fc0d511f4add.png)

Then you can rewrite the intro to not mention the names.